assets/css/_route_ladder.scss

```diff
@@ -22,14 +24,40 @@
 
   .c-route-ladder__dropdown-button {
     height: 2.25rem;
-    width: 2.25rem;
+    width: 1.8125rem;
```

Comment:
Can any of this be done with bootstrap utilities? I suspect the width can't, because bootstrap isn't flexible enough, but I'm thinking a bunch of the others can.

Reply:
Why bulk up a lot of inline styles via utilities if this className needs to exist anyway?

Reply:
I think the major advantage is that it reduces CSS bundle size, and is one of the major 

Caveat: I don't feel strongly about this, but I've gotten similar feedback in the past, and found the utilities to be quite nice once I wrapped my head around them. I'd be interested in @firestack's perspective on this too, because I think she understands this better than I do.

Reply:
I'm not _personally_ a fan of the tailwind-style-utility classes, but in the interest of lowering the barrier to entry of styling to Skate, and using common sizes from design, I do prefer using utility classes, as long as we're trying to use bootstrap.

The benefit of something like `ps-1`(padding start size 1) is that if we decide later that 1 should be bigger or smaller, it applies everywhere, instead of needing to track down every single instance.

Reply:
Ok, so in the future, I should go heavy on the inline utilities, and _additionally_ use CSS for further styling? I figured even having the styling class at all would bulk up the CSS bundle, but perhaps not.

Merging and adding this to our coffee chat list!